tpgtools/api/clouddeploy/samples/custom_target.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I don't immediately see what's wrong:

E0430 19:51:13.156675   24858 sample.go:202] failed to convert "custom.target.json"
F0430 19:51:13.158027   24858 sample.go:231] Error generating sample dependency custom.target.json: unexpected end of JSON input

Reply:
@rileykarson me too. Given the urgency of moving to 1.66 I deleted those new files. They have nothing to do with 1.66 and they can always go in later I guess..